hello-world/brysonsteck/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Unless you have a very specific reason, it's better to unify your .gitignore at the top level (of your submission).

You can ignore directories named `build` at any level using `build/` instead of `/build`